lessons/intro/cython/info.yml

```diff
@@ -3,3 +3,4 @@ title: Cython
 style: md
 attribution: Pro kurz MI-PYT na ČVUT napsali Petr Viktorin, Miro Hrončok a další, 2016-2017.
 license: cc-by-sa-40
+license_code: cc0
```

Comment:
The C API example is based on Python docs ([a](https://docs.python.org/3/extending/extending.html), [b](https://docs.python.org/3/extending/building.html#building-c-and-c-extensions-with-distutils)), and on files like [xxlimited.c](https://github.com/python/cpython/blob/master/Modules/xxlimited.c). We might want to push relicencing these first.

Reply:
Removed for now.